_data/team.yml

```diff
@@ -123,3 +123,14 @@ groups:
           - {name: "google-plus", link: "https://plus.google.com/117970536994604205384"}
           - {name: "site", link: "http://swarm.cs.pub.ro/~razvan/wiki/"}
           - {name: "linkedin", link: "https://www.linkedin.com/in/razvandeaconescu"}
+
+      - id: 8
+        name: "Paula Mădălina"
+        surname: "Crîșmaru"
+        title: ""
```

Comment:
Please add information for the following parameters:

- `title`

- `company`

- `rockstar`

- `bio`